extensions/admin-pack/package.json

```diff
@@ -9,6 +9,12 @@
 		"vscode": "*",
 		"azdata": "*"
 	},
+	"capabilities": {
+		"virtualWorkspaces": false,
```

Comment:
Do we need to set virtualWorkspaces as well? Reading docs on it now.

Reply:
I didn't see it enabled on most the built-in vscode packages, so unlikely we need it.  I'm planning to merge this later tonight so we can validate the build tomorrow.  If it ends up being needed for some reason we can add that flag is a separate PR.